guided-demo/graphs-1/README.md

```diff
@@ -33,3 +33,16 @@ This is the recommended approach for our graph projects.
 ## Comparison
 
```

Comment:
Very interesting. You get many extra credit points for this. Keep up the great work